client/job.lua

```diff
@@ -122,6 +123,7 @@ RegisterNetEvent('QBCore:Client:OnPlayerLoaded', function()
         QBCore.Functions.GetPlayerData(function(PlayerData)
             PlayerJob = PlayerData.job
             onDuty = PlayerData.job.onduty
+            SetEntityHealth(PlayerPedId(), PlayerData.metadata["health"])
             SetPedArmour(PlayerPedId(), PlayerData.metadata["armor"])
             if (not PlayerData.metadata["inlaststand"] and PlayerData.metadata["isdead"]) then
```

Comment:
You can use `ped` here instead of `PlayerPedId()`

Reply:
Does it matter, or is it just normal practice?

Reply:
No need to re-call that function when it's already assigned to a variable

Reply:
yeah I see what you mean, it's already localised at the top of the function.